pages/dkp/kommander/2.2/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -72,6 +72,10 @@ To attach an existing EKS cluster, refer to the specific information in [Attach
 Consider the additional resource requirements for running the platform services you want Kommander to manage, and ensure that your existing clusters comply.
 -->
 
+## Attach clusters with an existing cert-manager installation
+
+If you are attaching clusters that already have cert-manager installed, the cert-manager HelmRelease provided by Kommander will fail to deploy, due to your existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this.
```

Comment:
```suggestion
If you are attaching clusters that already have cert-manager installed, the cert-manager HelmRelease provided by Kommander will fail to deploy, due to the existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this failure, it will have no impact on the operation of the cluster.
```